recipe/meta.yaml

```diff
@@ -7,19 +7,30 @@
 # git-annex into conda environments that include packages which conflict with some dependencies of git-annex.  The
 # standard (non-nodep) version is preferred; the nodep version should only be used in case of unresolvable dependency
 # conflicts.
+
+#
+# The standalone build is currently taken from the autobuilder run by git-annex' author, as ready-made binaries.
+# Better would be to build it as part of the conda-forge packaging process, but the current sources are designed to
+# be built on a Debian distribution, which conda-forge does not use.
+#
+
+#
+# The version number printed by the standalone (nodep) version is typically behind the version number of the standard version,
+# but in practice has the same functionality -- see discussion here:
+# https://git-annex.branchable.com/forum/standalone_tarballs_for_specific_versions/#comment-d15b771fb5d23ce0c8dfb7f1c25064c5
 #
+{% set version = "8.20210903" %}
+{% set version_printed = version %}  # [not nodep]
+{% set version_printed = "8.20210804" %}  # [nodep]
```

Comment:
There were some critical fixes since then, I wonder what is actual version of that build?

Reply:
actual version is a few commits before the standard build, from what I can tell insignificant ones